packages/web-twig/src/Resources/components/Header/README.md

```diff
@@ -34,7 +85,16 @@ With Html syntax lexer (enabled by default):
         />
     </a>
 </Header>
-<Header isSimple isInverted>
+```
+
+## Simple Header
+
+The `isSimple` modifier makes the header bar slightly shorter and aligns its
```

Comment:
is `shorter` the correct word? I am not sure

Reply:
I believe it's the right way to say "the opposite of tall" 🙂. Neither _smaller_ nor _lower_ do not seem to fit as good…